apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -172,6 +172,34 @@ public object Clone()
         /// <inheritdoc />
         public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancellationTokenSource.Token).ResultWithUnwrappedExceptions();
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings)
+        {
+            return ExecuteReaderAsync(singleUseReadSettings, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public async Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings, 
+            CancellationToken cancellationToken)
```

Comment:
however, this place cannot be changed because we are deriving from a base class that has the overload.  I changed a couple of other places however.